testing/checkin/runner/ops_io.rs

```diff
@@ -65,3 +69,41 @@ pub fn op_pipe_create(op_state: &mut OpState) -> (ResourceId, ResourceId) {
   });
   (rid1, rid2)
 }
+
+struct FileResource {
+  fd: deno_core::ResourceHandleFd,
+}
+
+impl FileResource {
+  fn new(file: tokio::fs::File) -> Self {
+    let fd = file.as_raw_fd();
+    Self { fd }
+  }
+}
+
+impl Resource for FileResource {
+  fn backing_fd(
```

Comment:
I think you'll want to override this one instead to make it work on unix and windows:

```
  fn backing_handle(self: Rc<Self>) -> Option<ResourceHandle> {

  }
```